### DISCUSSION TITLE
Trying to install enve Shader Plugin AxiomGraph for Friction. Getting error

### DISCUSSION THREAD

**Discussion by AlmostMikhail** [friction2d/friction#382] [Q&A]
I'm getting this error even if i install version 2. So, version 1 working fine with no errors. Just a heads up.
![1(2)](https://github.com/user-attachments/assets/fbdd7d19-91a8-4d3e-a739-60d2c2547489)
![Friction(2)](https://github.com/user-attachments/assets/146ab298-338b-4edc-8e13-8654c526587a)

**Comment by rodlie** [comment]
https://github.com/friction2d/friction-shader-plugins/tree/main/src

**Reply by JeetChaurasia** [reply]
brother we made a discord server for friction 2d. please join us. we would love to get in touch with you and will be able to give personal ideas and contribution. we already got some people in there who loves your software and would like to help, discuss and moree
